RELEASE.md

```diff
@@ -0,0 +1,53 @@
+# Boxo Releases
+
+## Release Policies
+### Versioning
+Boxo only releases minor and patch versions, and has no plans to change major versions from v0.
+
+Minor versions contain new or removed functionality, or significant changes in dependencies.
+
+Patch versions contain "fixes", which are generally small, targeted, and involve limited changes in functionality (such as bug fixes, security fixes, etc.). The amount of backporting depends on the severity of the issue and the impact on users...most non-critical fixes won't be backported.
+
+As a result, Boxo maintainers recommend that consumers stay up-to-date with Boxo releases.
+
+### Go Compatibility
+At any given point, the Go team supports only the latest two versions of Go released (see https://go.dev/doc/devel/release). Boxo maintainers will strive to maintain compatibilty with the older of the two supported versions, so that Boxo is also compatible with the latest two versions of Go.
+
+### Release Criteria
+Boxo does not follow a regular release cadence. Releases are performed on-demand as significant changes are made (new features added, refactorings, deprecations, etc.). The [release process](#release-process) below describes how maintainers perform a release.
+
+### Testing
+Boxo maintainers are in the process of moving tests into Boxo from Kubo. Until this is completed, Boxo releases rely on Kubo tests for extra confidence. So releases must include plumbing Boxo into Kubo and ensuring its tests pass. See [Release Process](#release-process) for details.
```

Comment:
We should have a tracking issue for this so someone can follow along and so it's clear when it's done.  (I'm not aware of one.  I can create it but I also don't know the done criteria here.). Lets link to that issue.

Reply:
added https://github.com/ipfs/boxo/issues/275